src/js/pages/timer.js

```diff
@@ -0,0 +1,28 @@
+
+import { Howl } from '../libs/howler.js';
+
+  let sound = new Howl({
+   src: ['song.mp3'],
```

Comment:
чтобы в сборку попал этот файлик, нужно подключить его через `import pathToMp3 from './song.mp3'`
и в этой строке передавать путь из импорта
```suggestion
   src: [pathToMp3],
```